net/Socket.hpp

```diff
@@ -1117,15 +1117,18 @@ class StreamSocket : public Socket,
                 len = readData(buf, sizeof(buf));
                 last_errno = errno;
 
-                LOG_TRC('#' << getFD() << ": Read incoming data " << len << " bytes in addition to "
-                            << _inBuffer.size() << " buffered bytes ("
-                            << Util::symbolicErrno(last_errno) << ": " << std::strerror(last_errno)
-                            << ')');
-
-                if (len < 0 && last_errno != EAGAIN && last_errno != EWOULDBLOCK)
-                    LOG_SYS_ERRNO(last_errno, '#' << getFD() << ": Socket read returned " << len);
-            }
-            while (len < 0 && last_errno == EINTR);
+                if (len < 0)
```

Comment:
I really think EAGAIN and EWOULDBLOCK are too noisy to log but ... if you need them for a bit - why not.

Reply:
We did log them previously. This reduces the total log entries, actually.